index.js

```diff
@@ -199,6 +176,7 @@ nabs.process = function process(tasks) {
   });
 
   log.info('Building scripts...');
+
```

Comment:
Please don't add extra whitespace (unless there's a reason I'm not aware of?) Thanks!

Reply:
Sorry, I checked this with two different editors and there's no extra whitespace there. It's a new line, do we have a problem with a new line?

Reply:
@artlogic 
I removed the line break to keep the code style.